### ISSUE TITLE
Unable to uninstall decrapifyYahooFF from Greasemonkey 4

### ISSUE THREAD

**Issue by jcosentino** [greasemonkey/greasemonkey#2664]
When I go to uninstall, it loads the countdown (10) but it does not countdown. I have removed other scripts with no problem.
See:
decrapifyYahooFF
https://openuserjs.org/scripts/the_umm_guy/decrapifyYahooFF
https://www.reddit.com/r/fantasyfootball/comments/53jwyq/i_created_a_user_script_for_tampermonkey_to/

Firefox 57.0.0 Windows 10 x64 installation
Locale: es-ES
Greasemonkey 4.0

**Comment by jcosentino** [comment]
I removed Greasemonkey and the scripts were erased. I re-added the script to test if I can remove it normally. Removal now works fine!

**Comment by Sxderp** [comment]
While your issue might be resolved, the fact that it happened may point to some other issue. Unfortunately could be difficult to pin down.

**Comment by jcosentino** [comment]
I upgraded my installation of Firefox 56.0.2 to 57.0.0 final through an installable from the Firefox ftp site. I had Greasemonkey 3.x installed and that was automatically updated to 4. I did not have the script installed; I removed it. However, I did update Greasemonkey to some 4.0 development version in Firefox 56.x to test it out. At that point I may have had this script installed. But then I downgraded Greasemonkey to 3.x since I like having the latest stable versions of things. THEN I believe I removed the script. So maybe there was some lingering configuration somewhere of the 4.0 that thought that the script was still there?
